readml/main.py

```diff
@@ -1,14 +1,22 @@
+import json
 from pprint import pformat
+from typing import Dict, Optional
 
 import click
-import sklearn
 
 from readml.explainers.core import interpret_dl, interpret_ml
 from readml.logger import logger
 from readml.utils import _parse_and_check_config
 
 
 @click.command()
+@click.option(
+    "--config-values",
+    default=None,
+    show_default=True,
+    metavar="",
+    help="Dictionnary of configuration adapted to the type of problematic you need to interpret",
```

Comment:
Add that templates are available in `readml/config/` folder

Reply:
Done :)